beetle-core/src/main/java/com/xing/beetle/dedup/api/MessageListener.java

```diff
@@ -33,21 +30,7 @@ default void onDropped(M message, String reason) {
     logger().log(Level.WARNING, reason);
   }
 
-  default void onRequeued(M message) throws IOException {
-    if (message instanceof Message) {
-      Message m = (Message) message;
-      logger().log(Level.WARNING, m.getMessageProperties().getMessageId() + " requeued");
-      return;
-    }
-
-    if (message instanceof Delivery) {
-      Delivery m = (Delivery) message;
-      logger().log(Level.WARNING, m.getProperties().getMessageId() + " requeued");
-      return;
-    }
-
-    logger().log(Level.WARNING, message + " requeued");
-  }
+  default void onRequeued(M message) throws IOException {}
```

Comment:
I am wondering if we shouldn't keep these log statments but maybe on `debug` or even `trace` level.

Reply:
I'm not sure if we really need to log them because they are actually requeue attempts. The real requeue should be logged instead. I will check if we do so.